packages/theme/readme.md

```
@@ -0,0 +1,215 @@
+
+export default {
+  extends: DefaultTheme,
+  enhanceApp() {}
+} satisfies Theme;
+```
+
+## Contributors
+
+Please read [contributing guidelines here](./CONTRIBUTING.md).
+
+<a href="https://github.com/tsedio/tsed/graphs/contributors"><img src="https://opencollective.com/tsed/contributors.svg?width=890" /></a>
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)